app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,42 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ $govuk-images-path: "/assets/";
+ $govuk-fonts-path: "/assets/";
+ $govuk-font-url-function: 'font-url';
+ $govuk-image-url-function: 'image-url';
+
+ $govuk-global-styles: true;
+
+ @import "govuk-frontend/all";
+ 
+#clear-search-div {
```

Comment:
**[not a blocker/nit pick]** The styles starting here seem to be specific to the `provider` frontend, they should probably be added when we add its functionality here.